data/faq20.toml

```diff
@@ -65,6 +65,8 @@ a = """
 The target metric value is used by the Horizontal Pod Autoscaler (HPA) to make scaling decisions.
 
 The current target value on the Horizontal Pod Autoscaler (HPA) often does not match with the metrics on the system you are scaling on. This is because of how the Horizontal Pod Autoscaler's (HPA) [scaling algorithm](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#algorithm-details) works.
+
+If you are checking your metric directly in the HPA, take in consideration that KEDA currently only supports average metrics, so the value that you will see in the HPA is the average of the metric between the total amount of pods.
```

Comment:
What about this?
```suggestion
KEDA currently only supports average metrics. This means that the HPA will use the average 
value of the metric between the total amount of pods.
```

Reply:
So this is saying that it will use `average(metric) / # of pods`; is that correct?

Reply:
yes

Reply:
if you mean `targetValue = metric/ # of pods` yes

Reply:
Basically if the upstream metrics is 10 and you have 5 instances, the value that you will see in the HPA is 2 (10/5)

Reply:
is there any reason for the new line between average and value or it's just a typo?